common/utils.js

```diff
@@ -0,0 +1,4 @@
+let UNIQUE_ID_COUNTER = 0;
+export function getUniqueString (prefix = 'dt') {
```

Comment:
huh, I would prefer to have this like `dt-unique`. Just because maybe `dt1` or `dt2` is more probably to be used.

Reply:
That's just the default prefix, for the icons I'm using `dt-icon` prefix and we're currently just using this util on icons, once we move the local usage of dialtone to this everything would be ok, I don't see any problem.